test/Altinn.Platform.Authentication.Tests/Mocks/SystemRegisterServiceMock.cs

```diff
@@ -141,5 +141,10 @@ public Task<bool> CreateClient(string clientId, CancellationToken cancellationTo
         {
             throw new NotImplementedException();
         }
+
+        public Task<Guid?> CreateRegisteredSystem(RegisteredSystem system, CancellationToken cancellation = default)
```

Comment:
will this be a new PR after we merge pre approve branch?

Reply:
yes